MaterialComponentsBeta.podspec

```diff
@@ -66,15 +66,15 @@ Pod::Spec.new do |mdc|
     extension.dependency "MaterialComponents/schemes/Typography"
   end
 
-  mdc.subspec "BottomNavigation" do |component|
+  mdc.subspec "BottomNavigationBeta" do |component|
```

Comment:
Why are we putting "Beta" in the component name?

Reply:
This is a WIP so it's not ready for review.